doc/sidewalk_known_issues.rst

```diff
@@ -8,37 +8,44 @@ See the list of known issues that are valid for the current state of development
+    The error reproduces rarely - once per few days.
+    In the release mode the device resets automatically, however, in the debug mode it need to be reset manually.
+    Currently, this issue occurs for Sidewalk 1.14 libraries and it will be fixed in the version 1.16.
 
-* KRKNWK-17320: Bluetooth LE connection between Gateway and End Node drops within 30 seconds of being idle.
-    The issue occurs only on the nRF5340 development kit (experimental support) and happens due to failing certification test.
+* KRKNWK-17320: Bluetooth LE connection between Gateway and End Node drops within 30 seconds of being idle
+    The issue happens due to failing certification test.
 
-* KRKNWK-17321: The device fails after multiple restarts.
-   The issue occurs only on the nRF5340 development kit (experimental support) and happens due to failing certification test.
+    **Affected platforms:** nRF5340
 
-* KRKNWK-17308: Release version does not boot.
-    The issue occurs only on the nRF5340 development kit (experimental support).
+* KRKNWK-17321: The device fails after multiple restarts on the nRF5340 DK.
+   The issue happens due to failing certification test.
 
-* KRKNWK-17114: Bootloader configuration is not optimized.
-   The issue occurs only on the nRF5340 development kit (experimental support).
+   **Affected platforms:** nRF5340
+
+* KRKNWK-17308: Release version does not boot on the nRF5340 DK.
+
+   **Affected platforms:** nRF5340
+
+* KRKNWK-17114: Bootloader configuration is not optimized on the nRF5340 DK.
    Due to large bootloader partition size (48 kB), the DFU always requires an external flash.
 
-* KRKNWK-17035: Sensor monitor uplink messages are lost when the notification period is longer than 30 seconds.
+   **Affected platforms:** nRF5340
+
+* KRKNWK-17035: Sensor monitor uplink messages are lost when the notification period is longer than 30 seconds
     If the notification period is set to longer than 30 seconds, the sensor monitor uplink messages are lost.
     For this reason, the notification period is set to 15 seconds by default.
 
-* KRKNWK-14583: Bus fault after flash, before the :file:`Nordic_MFG.hex` data flash.
+* KRKNWK-14583: Bus fault after flash, before the :file:`Nordic_MFG.hex` data flash
     For sub-GHz samples, when the :file:`Nordic_MFG.hex` file is missing, the device throws a hard fault during initializing the Sidewalk stack.
     Proper error handling will be implemented, but the temporary solution is to write manufacturing hex to the device and reset it.
 
-* KRKNWK-14299: NRPA MAC address cannot be set in Zephyr.
+* KRKNWK-14299: NRPA MAC address cannot be set in Zephyr
     The non-resolvable private address (NRPA) cannot be set in the connectable mode for Bluetooth LE.
     Currently there is no workaround for this issue.
 
-* Dynamic FSK-WAN Profile switching from 2 to 1 is prohibited.
+* Dynamic FSK-WAN Profile switching from 2 to 1 is prohibited
```

Comment:
Good point. The reason it is different from "normal" Nordic issues, is this issue is a Sidewalk 1.14 release limitation. I'm not sure how we should handle in in NCS styled documentation

Reply:
Got it! I added a line below explaining that it's a limitation. As long as the issue gets a jira number I think it can be treated like any other item in the ncs known issues page

Reply:
FYI: this limitation will not be moved to ncs, we will create a page within Sidewalk documentation with all the links to ncs and known issues, and we'll include a limitations subsection where this will be incorporated (this solution has been implemented before).